buildkite/bazel-bench/bazel_bench.py

```diff
@@ -30,30 +31,26 @@
 import yaml
 
 
-def _platform_path_str(posix_path):
-    """Converts the path to the appropriate format for platform."""
-    if os.name == "nt":
-        return posix_path.replace("/", "\\")
-    return posix_path
-
-
 # TMP has different values, depending on the platform.
 TMP = tempfile.gettempdir()
 PROJECTS = [
```

Comment:
should this config rather live in a yaml file in the future? If so, please add a TODO.

Reply:
Done.